R/drop_na.r

```diff
@@ -0,0 +1,36 @@
+#' Drop rows containing missing values
+#'
+#' @param data A data frame.
+#' @param ... Specification of variables to consider while dropping rows.
+#'   If empty, consider all variables. Use bare variable names. Select all
+#'    variables between x and z with \code{x:z}, exclude y with \code{-y}.
+#'    For more options, see the \link[dplyr]{select} documentation.
+#' @seealso \code{\link{drop_na_}} for a version that uses regular evaluation
+#'   and is suitable for programming with.
+#' @examples
+#' library(dplyr)
+#' df <- data_frame(x = c(1, 2, NA), y = c("a", NA, "b"))
+#' df %>% drop_na()
+#' df %>% drop_na(x)
+#' @export
+drop_na <- function(data, ...) {
+  relevant_cols <- unname(dplyr::select_vars(colnames(data), ...))
+  drop_na_(data, relevant_cols)
+}
+
+#' Standard-evaluation version of \code{drop_na}.
+#'
+#'
+#' @param data A data frame.
+#' @param relevant_cols Character vector of column names. If empty or \code{NULL}, all
```

Comment:
RE `NULL`: more or less for convenience, so that one can write `drop_na_(data, NULL)` instead of `drop_na_(data, character())`

Re checking that it is a character vector: I didn't add that because e.g. `fill_.data.frame` also doesn't do this check. Should I add a commit which adds it there, too?